admin/src/scenes/settings/tabs/EligibilityTab.tsx

```diff
@@ -0,0 +1,182 @@
+              placeholder="JJ/MM/AAAA"
+              isTime={false}
+              value={values.bornBefore}
+              onChange={(e) => setValues({ ...values, bornBefore: e })}
+              error={error.bornBefore || ""}
+              readOnly={false}
+              disabled={isLoading || readOnly}
+            />
+          </div>
+        </div>
+        {error?.interval && <div className="text-[#EF4444]">{error?.interval}</div>}
+      </div>
+      <div className="flex flex-col gap-8 rounded-xl bg-white px-8 pb-12 pt-8 shadow-[0_8px_16px_0_rgba(0,0,0,0.05)]">
+        <div className="flex w-full flex-col gap-8">
+          <p className="text-lg font-medium leading-5 text-gray-900">Situation scolaire</p>
+          <div className="flex row w-full gap-4 justify-center">
+            {Object.keys(GRADES).map((grade, index) => (
+              <div key={grade + "grade" + index} className="flex h-6 items-center">
+                <input
+                  id={grade}
+                  name={grade}
+                  type="checkbox"
+                  disabled={isLoading || readOnly}
+                  checked={values.schoolLevels.includes(grade)}
+                  className="h-4 w-4 rounded border-gray-300 text-blue-600 cursor-pointer"
+                  onChange={(e) => {
+                    const newValues = e.target.checked ? [...values.schoolLevels, grade] : values.schoolLevels.filter((g) => g !== grade);
+                    setValues((prev) => ({ ...prev, schoolLevels: newValues }) as EligibilityTabsState);
+                  }}
+                />
+                <label htmlFor={grade} className="mt-1.5 ml-2 text-sm cursor-pointer whitespace-nowrap">
+                  {translateGrade(grade)}
+                </label>
+              </div>
+            ))}
+          </div>
+          {error.schoolLevels && <div className="text-[#EF4444]">{error.schoolLevels}</div>}
+        </div>
+      </div>
+      <div className="flex flex-col gap-8 rounded-xl bg-white px-8 pb-12 pt-8 shadow-[0_8px_16px_0_rgba(0,0,0,0.05)]">
+        <div className="flex w-full flex-col gap-8">
+          <p className="text-lg font-medium leading-5 text-gray-900">Eligibilité par départements</p>
+
+          <div className="grid grid-cols-2 gap-4">
+            {Object.keys(region2department).map((region, index) => (
+              <Select
+                key={region + "select" + index}
+                isMulti={true}
+                isClearable={true}
+                label={region}
+                disabled={isLoading || readOnly}
+                options={region2department[region].map((dep) => ({ label: dep, value: dep }))}
+                value={values?.zones?.filter((dep) => region2department[region].includes(dep)).map((dep) => ({ label: dep, value: dep }))}
+                onChange={(data) => {
+                  const newValues = data.map((option) => option.value);
+                  setValues(
+                    (prev) =>
```

Comment:
```suggestion
                    (prev: EligibilityTabsState) =>
```
ça devrait éviter le `as`